### DISCUSSION TITLE
How to prevent LTeX from spamming my "PROBLEMS" tab

### DISCUSSION THREAD

**Discussion by matthieubraun** [valentjn/vscode-ltex#838] [General]
I am using the vscode extension LTeX to help me find misspell in my LaTeX documents. But there is one thing that annoys me, it also finds "misspells" in some latex configuration files. How can I prevent LTeX from spamming the "PROBLEMS" tab in vscode like pic related ? [pic of the problems](https://i.stack.imgur.com/1wkO5.png)

I tried to look the extension settings tab of LTeX. I was expecting to find a settings to prevent it from outputting in the "PROBLEMS" tab

**Comment by matthieubraun** [comment]
Maybe add a add a filter, include and exclude, for the reported problems ?

**Reply by hittjwX** [reply]
Yes, it seems when I do a large search and replace LTeX checks every file. It slows things down a lot. It would be nice for a setting to only check files the user opens to see in the editor. I've set `Check Frequency` to manual so I can get some work done.

**Comment by guraltsev** [comment]
In the extension settings

![image](https://user-images.githubusercontent.com/7040974/236642765-d195abe1-ba36-4fd1-b8ed-42868b8ca9a6.png)

you can set ```"ltex.diagnosticSeverity": "hint"```
